doc_admin/AdminManual.adoc

```diff
@@ -1077,21 +1146,26 @@ see <<log4j-properties,log4j.properties>>) and try to use the SetupGSM utility (
 (see <<folders,Folders>>).
 
 
-===== 5.1.23.2 SetupGSM
+===== 3.1.23.2 SetupGSM utility
+
+To setup GSM communication it is possibile use the included SetupGSM utility and follow these instructions:
```

Comment:
spelling: `possibile` -> 'possible`

Reply:
change `possible use` to `possible to use`

Reply:
Done, thank you !